distribution/changelog.txt

```diff
@@ -4,10 +4,16 @@
 - Feature: [#20709] [Plugin] Plugins can now check metadata from all registered plugins.
 - Feature: [#21376] Add option to reload an object (for object developers).
 - Feature: [#21455] Add option to control hidden scenery.
+- Feature: [objects#282] Added RCT1 beta terrains.
 - Improved: [#21356] Resize the title bar when moving between displays with different scaling factors on Windows systems.
 - Improved: [#21388] Tooltips will now show even when an error message is present.
 - Improved: [#21423] Add mechanism to allow building partly underground.
+- Improved: [objects#272, objects#276, objects#277, objects#278, objects#279, objects#280, objects#281] Add missing RCT1 ride and vehicle objects.
+- Improved: [objects#275, objects#284, objects#286] Add missing RCT1 walls.
+- Change: [#21350] Group Dodgems and Flying Saucers vehicles.
```

Comment:
I am trying to figure out what this does.
I can't select the vehicles on the flying saucers/dodgems when both are available, unless that's not what you meant with this.

Reply:
It is not, no. Rather, try selecting all the different dodgems vehicles in the Object Selection and check how they appear in the New Ride window when grouping is enabled.

Reply:
I see. They group seperately. I got confused with the wording.
Though right now I still see some versions separated.
![image](https://github.com/OpenRCT2/OpenRCT2/assets/1516893/b2c52996-661f-497a-b56d-0705a7a2257b)
![image](https://github.com/OpenRCT2/OpenRCT2/assets/1516893/0329cc1a-d254-46ea-9139-159cf9df44be)

Reply:
Ah I see. I got confused by the wording. I thought you merged Dodgems & Flying Saucers with eachother (which could be possible because they're basically the same ride on a different platform). But they're still seperate, just the list is made smaller, which is good.